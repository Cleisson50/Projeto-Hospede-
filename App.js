import React from "react";
import { NavigationContainer } from "@react-navigation/native";
import { createStackNavigator } from "@react-navigation/stack";
import Task from "./src/pages/Task/";
import Login from "./src/pages/Login";
import NewUser from "./src/pages/NewUser";
import RedefinirSenha from "./src/pages/Redefinir";

const Stack = createStackNavigator();

export default function App() {
  return (
    <NavigationContainer>
      <Stack.Navigator initialRouteName="Login">
        <Stack.Screen name="Login" component={Login} options={{ title:"Hospede+" }} />
        <Stack.Screen name="NewUser" component={NewUser} options={{headerLeft: null, title:"Hospede+" }} />
        <Stack.Screen name="Task" component={Task} options={{ headerTintColor: "#f92e6a", headerLeft: null, title:"Hospede+" }} />
        <Stack.Screen name="RedefinirSenha" component={RedefinirSenha} options={{headerLeft: null, title:"Hospede+"}} />
      </Stack.Navigator>
    </NavigationContainer>
  );
}