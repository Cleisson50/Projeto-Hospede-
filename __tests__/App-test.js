import renderer from 'react-test-renderer';
import App from '../App';
import { fireEvent, render } from '@testing-library/react-native';
jest.mock("react-native/Libraries/Animated/NativeAnimatedHelper");

test('teste de snapshot', () => {
    // const tela = renderer.create(<App />).toJSON();
    // expect(tela).toMatchSnapshot();
});

test('teste de navegaÃ§Ã£o', () => {

});