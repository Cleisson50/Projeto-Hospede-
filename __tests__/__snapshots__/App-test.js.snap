// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste de Snapshot Snapshot de App 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "rgb(242, 242, 242)",
            "flex": 1,
          },
          undefined,
        ]
      }
    >
      <View
        collapsable={false}
        pointerEvents="box-none"
        style={
          Object {
            "zIndex": 1,
          }
        }
      >
        <View
          accessibilityElementsHidden={false}
          importantForAccessibility="auto"
          onLayout={[Function]}
          pointerEvents="box-none"
          style={null}
        >
          <View
            collapsable={false}
            pointerEvents="box-none"
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "zIndex": 0,
              }
            }
          >
            <View
              collapsable={false}
              style={
                Object {
                  "backgroundColor": "#4682B4",
                  "borderBottomColor": "rgb(216, 216, 216)",
                  "flex": 1,
                  "shadowColor": "rgb(216, 216, 216)",
                  "shadowOffset": Object {
                    "height": 0.5,
                    "width": 0,
                  },
                  "shadowOpacity": 0.85,
                  "shadowRadius": 0,
                }
              }
            />
          </View>
          <View
            collapsable={false}
            pointerEvents="box-none"
            style={
              Object {
                "height": 110,
                "maxHeight": undefined,
                "minHeight": undefined,
                "opacity": undefined,
                "transform": undefined,
              }
            }
          >
            <View
              pointerEvents="none"
              style={
                Object {
                  "height": 0,
                }
              }
            />
            <View
              pointerEvents="box-none"
              style={
                Object {
                  "alignItems": "stretch",
                  "flex": 1,
                  "flexDirection": "row",
                }
              }
            >
              <View
                collapsable={false}
                pointerEvents="box-none"
                style={
                  Object {
                    "alignItems": "flex-start",
                    "flexBasis": 0,
                    "flexGrow": 1,
                    "justifyContent": "center",
                    "marginStart": 0,
                    "opacity": 1,
                  }
                }
              />
              <View
                collapsable={false}
                pointerEvents="box-none"
                style={
                  Object {
                    "justifyContent": "center",
                    "marginHorizontal": 16,
                    "maxWidth": 718,
                    "opacity": 1,
                  }
                }
              >
                <Text
                  accessibilityRole="header"
                  aria-level="1"
                  collapsable={false}
                  numberOfLines={1}
                  onLayout={[Function]}
                  style={
                    Object {
                      "color": "#fff",
                      "fontSize": 22,
                      "fontWeight": "bold",
                    }
                  }
                >
                  Hospede+
                </Text>
              </View>
              <View
                collapsable={false}
                pointerEvents="box-none"
                style={
                  Object {
                    "alignItems": "flex-end",
                    "flexBasis": 0,
                    "flexGrow": 1,
                    "justifyContent": "center",
                    "marginEnd": 0,
                    "opacity": 1,
                  }
                }
              />
            </View>
          </View>
        </View>
      </View>
      <RNSScreenContainer
        onLayout={[Function]}
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <RNSScreen
          activityState={2}
          collapsable={false}
          forwardedRef={[Function]}
          pointerEvents="box-none"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        >
          <View
            collapsable={false}
            style={
              Object {
                "opacity": 1,
              }
            }
          />
          <View
            accessibilityElementsHidden={false}
            closing={false}
            gestureVelocityImpact={0.3}
            importantForAccessibility="auto"
            onClose={[Function]}
            onGestureBegin={[Function]}
            onGestureCanceled={[Function]}
            onGestureEnd={[Function]}
            onOpen={[Function]}
            onTransition={[Function]}
            pointerEvents="box-none"
            style={
              Array [
                Object {
                  "display": "flex",
                  "overflow": undefined,
                },
                Object {
                  "bottom": 0,
                  "left": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                },
              ]
            }
            transitionSpec={
              Object {
                "close": Object {
                  "animation": "spring",
                  "config": Object {
                    "damping": 500,
                    "mass": 3,
                    "overshootClamping": true,
                    "restDisplacementThreshold": 10,
                    "restSpeedThreshold": 10,
                    "stiffness": 1000,
                  },
                },
                "open": Object {
                  "animation": "spring",
                  "config": Object {
                    "damping": 500,
                    "mass": 3,
                    "overshootClamping": true,
                    "restDisplacementThreshold": 10,
                    "restSpeedThreshold": 10,
                    "stiffness": 1000,
                  },
                },
              }
            }
          >
            <View
              collapsable={false}
              pointerEvents="box-none"
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                collapsable={false}
                forwardedRef={[Function]}
                needsOffscreenAlphaCompositing={false}
                onGestureHandlerEvent={[Function]}
                onGestureHandlerStateChange={[Function]}
                style={
                  Object {
                    "flex": 1,
                    "transform": Array [
                      Object {
                        "translateX": 0,
                      },
                      Object {
                        "translateX": 0,
                      },
                    ],
                  }
                }
              >
                <View
                  style={
                    Array [
                      Object {
                        "flex": 1,
                        "overflow": "hidden",
                      },
                      Array [
                        Object {
                          "backgroundColor": "rgb(242, 242, 242)",
                        },
                        undefined,
                      ],
                    ]
                  }
                >
                  <View
                    style={
                      Object {
                        "flex": 1,
                        "flexDirection": "column-reverse",
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "flex": 1,
                        }
                      }
                    >
                      <View
                        style={
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "#EEE9E9",
                            "flex": 1,
                            "justifyContent": "center",
                          }
                        }
                      >
                        <Text
                          style={
                            Object {
                              "color": "#000000",
                              "fontSize": 48,
                              "fontWeight": "bold",
                              "marginBottom": 80,
                            }
                          }
                        >
                          Login
                        </Text>
                        <TextInput
                          onChangeText={[Function]}
                          placeholder="Insira seu email"
                          style={
                            Object {
                              "borderBottomColor": "#000000",
                              "borderBottomWidth": 1,
                              "color": "#4d5156",
                              "height": 50,
                              "marginLeft": "auto",
                              "marginRight": "auto",
                              "marginTop": 10,
                              "padding": 10,
                              "width": 300,
                            }
                          }
                          testID="email"
                          type="text"
                          value=""
                        />
                        <View
                          style={
                            Object {
                              "alignItems": "center",
                              "borderBottomColor": "#000000",
                              "borderBottomWidth": 1,
                              "color": "#4d5156",
                              "flexDirection": "row",
                              "height": 50,
                              "marginLeft": "auto",
                              "marginRight": "auto",
                              "marginTop": 10,
                              "padding": 10,
                              "width": 300,
                            }
                          }
                        >
                          <TextInput
                            onChangeText={[Function]}
                            placeholder="Insira uma senha"
                            secureTextEntry={true}
                            style={
                              Object {
                                "color": "#4d5156",
                                "height": 50,
                                "marginLeft": "auto",
                                "marginRight": "auto",
                                "padding": 10,
                                "width": 265,
                              }
                            }
                            testID="senha"
                            type="text"
                            value=""
                          />
                          <View
                            accessible={true}
                            collapsable={false}
                            focusable={true}
                            onClick={[Function]}
                            onResponderGrant={[Function]}
                            onResponderMove={[Function]}
                            onResponderRelease={[Function]}
                            onResponderTerminate={[Function]}
                            onResponderTerminationRequest={[Function]}
                            onStartShouldSetResponder={[Function]}
                            style={
                              Object {
                                "marginLeft": 10,
                                "opacity": 1,
                              }
                            }
                          >
                            <Text />
                          </View>
                        </View>
                        <View />
                        <View
                          accessibilityState={
                            Object {
                              "disabled": true,
                            }
                          }
                          accessible={true}
                          collapsable={false}
                          focusable={false}
                          onClick={[Function]}
                          onResponderGrant={[Function]}
                          onResponderMove={[Function]}
                          onResponderRelease={[Function]}
                          onResponderTerminate={[Function]}
                          onResponderTerminationRequest={[Function]}
                          onStartShouldSetResponder={[Function]}
                          style={
                            Object {
                              "alignItems": "center",
                              "backgroundColor": "#4682B4",
                              "borderRadius": 50,
                              "flexDirection": "row",
                              "height": 50,
                              "justifyContent": "center",
                              "marginTop": 30,
                              "opacity": 1,
                              "width": 300,
                            }
                          }
                        >
                          <Text
                            style={
                              Object {
                                "color": "#ffffff",
                                "fontSize": 16,
                                "fontWeight": "bold",
                              }
                            }
                          >
                            Login
                          </Text>
                        </View>
                        <View
                          accessible={true}
                          collapsable={false}
                          focusable={true}
                          onClick={[Function]}
                          onResponderGrant={[Function]}
                          onResponderMove={[Function]}
                          onResponderRelease={[Function]}
                          onResponderTerminate={[Function]}
                          onResponderTerminationRequest={[Function]}
                          onStartShouldSetResponder={[Function]}
                          style={
                            Object {
                              "alignItems": "center",
                              "backgroundColor": "#4682B4",
                              "borderRadius": 50,
                              "flexDirection": "row",
                              "height": 50,
                              "justifyContent": "center",
                              "marginTop": 30,
                              "opacity": 1,
                              "width": 300,
                            }
                          }
                        >
                          <Image
                            source={1}
                            style={
                              Object {
                                "borderRadius": 6,
                                "height": 30,
                                "width": 30,
                              }
                            }
                            testID="Logogoogle"
                          />
                          <View>
                            <Text
                              style={
                                Object {
                                  "color": "#ffffff",
                                  "fontSize": 16,
                                  "fontWeight": "bold",
                                  "marginLeft": 10,
                                }
                              }
                            >
                              Login com Google
                            </Text>
                          </View>
                        </View>
                        <Text
                          style={
                            Object {
                              "color": "#000000",
                              "fontSize": 16,
                              "marginTop": 20,
                            }
                          }
                        >
                          Não está registrado?
                          <Text
                            onPress={[Function]}
                            style={
                              Object {
                                "color": "#0000CD",
                                "fontSize": 16,
                                "marginLeft": 10,
                              }
                            }
                            testID="novousuario"
                          >
                            Inscreva-se agora
                          </Text>
                        </Text>
                        <Text
                          style={
                            Object {
                              "color": "#000000",
                              "fontSize": 16,
                              "marginTop": 20,
                            }
                          }
                        >
                          Esqueceu sua senha?
                          <Text
                            onPress={[Function]}
                            style={
                              Object {
                                "color": "#0000CD",
                                "fontSize": 16,
                                "marginLeft": 10,
                              }
                            }
                            testID="novasenha"
                          >
                            Redefinir
                          </Text>
                        </Text>
                      </View>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </RNSScreen>
      </RNSScreenContainer>
    </View>
  </RNCSafeAreaProvider>
</View>
`;
